mscr-ui/src/modules/crosswalk-editor/mappings-accordion/index.tsx

```diff
@@ -376,18 +376,11 @@ function returnPath(id: string, label: string, schemaFormat: Format | undefined,
       }
     }
     return returnString;
-  } else {
+
     let className = '';
-    if (schemaFormat === Format.Rdfs || schemaFormat === Format.Owl) {
-      if (id.lastIndexOf('#') === -1) {
-        returnString = id.substring(id.lastIndexOf('/') + 1);
-      } else {
-        className = id.substring(id?.lastIndexOf('/') + 1, label.lastIndexOf('#')) + ':';
-        let itemName = id.substring(id.lastIndexOf('#') + 1);
-        returnString = className + itemName;
-      }
     } else if (schemaFormat === Format.Shacl) {
```

Comment:
Check the indentation, seems like this whole block might have one extra indentation, since it should be level with the original if-statement.